components/brave_rewards/browser/rewards_service_impl.h

```diff
@@ -226,6 +245,7 @@ class RewardsServiceImpl : public RewardsService,
   void OnPublishersListSaved(ledger::LedgerCallbackHandler* handler,
                              bool success);
   void OnTimer(uint32_t timer_id);
+  void TriggerOnContentSiteUpdated();
```

Comment:
there's no implementation for this

Reply:
@ryanml One for you?

Reply:
@tmancey must be a mistake as I did not work with such a method, will remove

Reply:
done